datasize/bitsize.go

```diff
@@ -34,35 +34,35 @@ const (
 	// B is short for Byte
 	B = Byte
 
-	// Kilobyte represents 1024 bytes
-	Kilobyte = 1024 * Byte
-	// KB is short for Kilobyte
-	KB = Kilobyte
-
-	// Megabyte represents 1024 kilobytes
-	Megabyte = 1024 * Kilobyte
-	// MB is short for Megabyte
-	MB = Megabyte
-
-	// Gigabyte represents 1024 megabytes
-	Gigabyte = 1024 * Megabyte
-	// GB is short for Gigabyte
-	GB = Gigabyte
-
-	// Terabyte represents 1024 gigabytes
-	Terabyte = 1024 * Gigabyte
-	// TB is short for Terabyte
-	TB = Terabyte
-
-	// Petabyte represents 1024 terabytes
-	Petabyte = 1024 * Terabyte
-	// PB is short for Petabyte
-	PB = Petabyte
-
-	// Exabyte represents 1024 petabytes
-	Exabyte = 1024 * Petabyte
-	// EB is short for Exabyte
-	EB = Exabyte
+	// Kibibyte represents 1024 bytes
+	Kibibyte = 1024 * Byte
```

Comment:
I think this will be yet another confusion from user side. From pure scientific notation maybe you are right, from majority of users - confusion. I would leave Kilobyte (KB) == 1024.

Reply:
I'd rather leave it as kibibyte, the consumer will not be digging in code.

Reply:
Example from docker: https://github.com/docker/go-units/blob/master/size.go

Reply:
In user-facing UIs we may use approximations if we see fit, but for the core code, we should call things as they are.